### DISCUSSION TITLE
Porting to ghost cms

### DISCUSSION THREAD

**Discussion by uPagge** [Lednerb/bilberry-hugo-theme#331] [Ideas]
Thinking about porting your theme to ghost cms?  A lot of people go there with hugo and I would love to use your theme on Ghost

Just writing short articles on hugo is a pain.  But with ghost it's the other way around

**Comment by igor-baiborodine** [comment]
Not really. I think the easiest way to integrate Bilberry with Ghost CMS would be to implement an auxiliary shell script that will pull content from Ghost and transform it into markdown files. Then Hugo will use these markdown files to build the site. Also, see this thread: https://forum.ghost.org/t/a-few-questions-about-using-hugo-and-ghost/26202/2.

**Reply by uPagge** [reply]
Yes, I've already thought about it. But I'm afraid it can lead to some problems, but in any case it's probably worth a try.

It's just that this topic will really be convenient on Ghost, as articles on Ghost are convenient to write even from your phone :)
